### PR TITLE
Shift click range selection for wells

### PR DESCRIPTION
**Pull request by dominikl**
Enables shift+click range selection for wells and disables acquisition and preview tab if multiple images/wells are selected. (Follow-up to PR #3311)

### CODE REVIEW

File: components/insight/SRC/org/openmicroscopy/shoola/agents/metadata/editor/EditorUI.java [hunk 1]
```diff
@@ -317,8 +317,10 @@ void handleImageSelection()
         ImageData img = model.getImage();
         if (img == null) return;
         tabPane.setEnabledAt(ACQUISITION_INDEX, img.getId() > 0);
+        boolean multi = model.isMultiSelection();
         boolean preview = model.isPreviewAvailable();
-        tabPane.setEnabledAt(RND_INDEX, preview);
+        tabPane.setEnabledAt(RND_INDEX, preview && !multi);
+        tabPane.setEnabledAt(ACQUISITION_INDEX, !multi);
```
Comment (mtbc) [inline components/insight/SRC/org/openmicroscopy/shoola/agents/metadata/editor/EditorUI.java]: this doesn't render line 319 obsolete? (I really don't know)

**Comment by mtbc** [comment]
Functionality works fine so, apart from code question above, looking good. :+1:

**Comment by dominikl** [comment]
@mtbc That's true. I removed the obsolete line. I don't think a re-review is necessary because of that.

**Comment by mtbc** [comment]
We don't still need the `img.getId() > 0` condition?

**Comment by dominikl** [comment]
You're right, to be safe I added the ```img.getId() > 0``` again, although I can't see a case, in which this wouldn't be true.

**Comment by mtbc** [comment]
Thanks, code now looks good to me.

**Comment by jburel** [comment]
Travis is now green. Merging

**Pull request merged**